agent/android/build.gradle

```diff
@@ -59,8 +59,8 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 31
+        minSdkVersion 26
```

Comment:
Was the minSdkVersion bump needed? 26 is oddly high for a library and will break a lot of builds.